src/index.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import 'normalize-css/normalize.css';
 
 import './index.scss';
 import { App } from './App';
 import * as serviceWorker from './serviceWorker';
+import { reducer } from './store/reducer';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(reducer);
```

Comment:
I recommend using https://github.com/zalmoxisus/redux-devtools-extension
it aids development and debugging